CSS/introduction.css

```diff
@@ -39,7 +39,6 @@
     display: flex;
     align-items: center;
     justify-content: center;
```

Comment:
if this file is for your doc site, then use utility class here, then you have to right less code

Reply:
sure will change, thanks for suggestion